types/estree/estree-tests.ts

```diff
@@ -223,7 +223,7 @@ boolean = memberExpression.computed;
 
 // Declarations
 var functionDeclaration: ESTree.FunctionDeclaration;
-identifier = functionDeclaration.id;
+var identifierOrNull: ESTree.Identifier | null = functionDeclaration.id;
```

Comment:
`functionDeclaration.id = null` would be a better test, as the `identifierOrNull` test wouldn't fail even without the change to the declaration.